packages/core/lib/frame-context.tsx

```diff
@@ -0,0 +1,51 @@
+import React, {
+  createContext,
+  useContext,
+  useRef,
+  RefObject,
+  useMemo,
+  useCallback,
+} from "react";
+import { useResetAutoZoom } from "./use-reset-auto-zoom";
+
+interface FrameContextType {
+  frameRef: RefObject<HTMLDivElement | null>;
+  resetAutoZoom: ReturnType<typeof useResetAutoZoom>;
+}
```

Comment:
I think this context can be simplified to simply `frameRef`. This could be useful in other places throughout the app.

Bundling `resetAutoZoom` in this context feels duplicative, and it can just as easily be called inline with `frameRef` as needed.

Reply:
removed the duplicate reference to the resetAutoZoom